openapi.yaml

```
@@ -964,7 +964,120 @@ paths:
                             schema:
                                 $ref: '#/components/schemas/Error'
                             example: 
-                                errors: ["An unexpected error occured"]    
+                                errors: ["An unexpected error occured"]
+    /projects/{project_id}/collaborators:
+        get:
+            summary: 'List project collaborators'
+            operationId: listProjectCollaborators
+            tags:
+                - Projects
+            parameters:
+              - name: project_id
+                in: path
+                description: ID of the Project
+                required: true
+                schema:
+                    type: string
+                    format: id
+              - name: page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+              - name: per_page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+            responses:
+                '200':
+                    description: 'List of project collaborators'
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/ProjectCollaborator'
+                '400':
+                    description: 'Bad Request'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["Bad Request"]
+                '404':
+                    description: 'Not found'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["Record not found"]
+                '500':
+                    description: Unexpected error
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Error'
+                            example: 
+                                errors: ["An unexpected error occured"]
+    /projects/{project_id}/collaborators/invite:         
```

Can we change this to:

```suggestion
    /projects/{project_id}/collaborators/invitation:         
```

The difference is subtle, but this endpoint is not actually I inviting anyone, it's creating an invitation that can then be used elsewhere. I think we should leave `/invite` open for future use.

---

I like that suggestion too. Then were not using a verb here as well. Nice!

---

Maybe to make it less confusing and REST-like we just do Chad's suggestion above with POST `/projects/{project_id}/invitations`?